plugins/tactics/src/Ide/Plugin/Tactic/BindSites.hs

```diff
@@ -0,0 +1,55 @@
+module Ide.Plugin.Tactic.BindSites
+  ( BindSites ()
+  , bindSites
+  , getDefiningBindings
+  ) where
+
+import           Data.IntervalMap.FingerTree (IntervalMap, Interval (..))
+import qualified Data.IntervalMap.FingerTree as IM
+import qualified Data.Map as M
+import qualified Data.Set as S
+import           Development.IDE.GHC.Compat
+import           Development.IDE.GHC.Error
+import           Development.IDE.Types.Location
+import           NameEnv
+import           SrcLoc
+
+
+-- TODO(sandy): Consolidate this with LocalBindings
```

Comment:
It would be nice if you made a PR on ghcide for this

Reply:
Done (I think)

https://github.com/haskell/ghcide/pull/845